sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 # Release History
 
-## 5.1.0-beta.3 (Unreleased)
-
+## 5.2.0-beta.1 (Unreleased)
```

Comment:
Change it to ## 5.1.0-beta.3 again :(
also, don't forget to add a description of the changes

Reply:
Done.